### PR TITLE
add __ne__ to `Dimension` so you can `filter = Dimension('dim') != val`

### PR DESCRIPTION
**Pull request by dakra**

### CODE REVIEW

**Comment by fjy** [comment]
:+1:

File: pydruid/utils/filters.py [hunk 1]
```diff
@@ -73,6 +73,9 @@ def __init__(self, dim):
     def __eq__(self, other):
         return Filter(dimension=self.dimension, value=other)
 
+    def __ne__(self, other):
+        return ~Filter(dimension=self.dimension, value=other)
```
Comment (fjy) [inline pydruid/utils/filters.py]: does this use eventually end up using Druid's internal NotFilter?

Comment (xvrl) [inline pydruid/utils/filters.py]: it does implement the `__invert__` operator, so yes

**Comment by xvrl** [comment]
:+1:

**Pull request merged**